### PR TITLE
Finish supervisor add subjects to course

### PR DESCRIPTION
**Pull request by binhthanhnguyen**

### CODE REVIEW

File: app/views/rails_admin/main/add_subjects_to_course.html.erb [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <%= form_for @object, url: add_subjects_to_course_path, method: :post do |f| %>
       <%= hidden_field_tag "course[subject_ids][]", nil %>
       <% @subjects.each do |subject| %>
-          <tr>
+          <tr class="<%= "hidden" if @object.subjects.include?(subject) %>" >
```
Comment (trantube94) [inline app/views/rails_admin/main/add_subjects_to_course.html.erb]: indent

**Comment by minhld1709** [comment]
OK

**Comment by trantube94** [comment]
OK

**Comment by vietanhbkaaa** [comment]
OK

**Pull request merged**